engine/admin/logging/overview.md

```diff
@@ -89,6 +89,28 @@ json-file
 {% endraw %}
 ```
 
+## Configure the delivery mode of log messages from container to log driver
+
+Docker provides two modes for delivering messages from the container to the log driver since Docker 17.04:
```

Comment:
Please remove mentions of specific versions that are already very old and unsupported. :)